src/licensedcode/data/rules/make-human-exception.RULE

```diff
@@ -0,0 +1,11 @@
+MakeHuman output GPL exception
```

Comment:
Do not add a rule that is the same as the license. This is an error condition. Every license is automatically made in a rule.... so rules are for variants

Reply:
So ,This file is not needed .Right?

Reply:
correct

Reply:
and its yml buddy too.